pic-sure-auth-db/db/sql/V5__MIGRATE_TO_ACCESSRULE_JOINTABLE.sql

```diff
@@ -28,5 +28,4 @@ UPDATE access_rule ar
     JOIN accessRule_subRule ars ON ars.accessRule_id = ar.uuid
     SET ar.subAccessRuleParent_uuid = NULL; -- Remove circular dependency reference so it can be dropped
 
--- Step 4: Drop the column from the original table
-ALTER TABLE access_rule DROP COLUMN subAccessRuleParent_uuid;
\ No newline at end of file
+-- Step 4: Drop the column from the original table
```

Comment:
Can we confirm this has never been run anywhere important? Generally we should not be editing migration files

Reply:
This change already exist in the release branch. I don't know why it is showing it as part of the diff. https://github.com/hms-dbmi/pic-sure-auth-microapp/blob/eab9a0f61a42a0b80a2f85ba6982efb3ed270e4e/pic-sure-auth-db/db/sql/V5__MIGRATE_TO_ACCESSRULE_JOINTABLE.sql#L31-L31